cmake/Format.cmake

```diff
@@ -13,21 +11,18 @@ function(get_changed_files outvar)
     OUTPUT_VARIABLE ancestor_commit
     OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-  # Changed files that have been committed
   execute_process(
     COMMAND git diff --diff-filter=d --name-only ${ancestor_commit}...${current_branch}
     OUTPUT_VARIABLE committed_files
     OUTPUT_STRIP_TRAILING_WHITESPACE)
   separate_arguments(committed_files NATIVE_COMMAND ${committed_files})
 
-  # Unstaged files
```

Comment:
why are comments being removed?

Reply:
That was an accident. Comments unremoved on new commit